api/v0/cells/cells.proto

```diff
@@ -87,12 +88,12 @@ service CellService {
   rpc List(CellServiceListRequest) returns (CellServiceListResponse) {}
 }
 
-/// An Aurae cell is a name given to Linux control groups (cgroups) that also include
-/// a name, and special pre-exec functionality that is executed from within the same context
-/// as any executables scheduled.
+/// An Aurae cell is a name given to Linux control groups (cgroups) that also
```

Comment:
Nothing to do with this PR, but we need to make these double slashes and think about documenting a formatter. Is there an equivalent to rustfmt.toml for proto files?

Reply:
To clarify, I don't expect this to be done in this PR. I was just noting my thoughts.

Reply:
i don't know about a rustfmt.toml, but things like prettier will format protos.